src/components/ChangePlan/PlanCalculator.js

```diff
@@ -7,6 +7,25 @@ import queryString from 'query-string';
 import { extractParameter, getPlanFee } from '../../utils';
 import { useRouteMatch, Link } from 'react-router-dom';
 
+const NavigatorTabs = ({ tabs, pathType, selectedPlanType }) => {
+  return (
+    <nav className="tabs-wrapper">
+      <ul className="tabs-nav" data-tab-active="1">
```

Comment:
Here I imagine that `data-tab-active` would shift accordingly with type plan selected by url, if any.

Reply:
I didn't realize about this. I will check it 👍

Reply:
I think that data-tab-active is not necessary.

Reply:
I have to make changes to apply UI, so I can fix this there.

Reply:
👍